infra/terraform/ec2.tf

```diff
@@ -0,0 +1,52 @@
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
+
+resource "aws_instance" "ec2" {
+  ami = data.aws_ami.ubuntu.id
+  instance_type               = "t2.micro"
+  associate_public_ip_address = true
+  key_name                    = aws_key_pair.keypair.key_name
+  subnet_id                   = aws_subnet.subnet1.id
+  vpc_security_group_ids      = ["${aws_security_group.owtf_sg.id}"]
+  root_block_device {
+    delete_on_termination = true
+    encrypted             = true
+    volume_size           = "25"
+    volume_type           = "gp2"
+  }
+  metadata_options {
+    http_put_response_hop_limit = 2
+    http_tokens = "required"
+  }
+  tags = {
+    "Name" = "owtf_instance"
+  }
+  provisioner "local-exec" {
```

Comment:
I'd set up AWS SSM instead of relying on SSH. It is much more secure and removes the need to expose port 22 to the internet.